settings.bash

```diff
@@ -18,9 +18,10 @@ export DOCKER_REGISTRY_AUTOPULL=false
 
 ### The default release mode to use if no mode paramater is given to ./exec.bash or ./stop.bash
 ### The checked in version should reflect the image status and be the highest availale image (base - devel - release)
-export DEFAULT_EXECMODE="base" # Use this only for setting up the initial devel image (modify setup_workspace.bash)
-# export DEFAULT_EXECMODE="devel" # This is used while deveoping code and preparing a relase
-# export DEFAULT_EXECMODE="release" # use the release as default
+#export DEFAULT_EXECMODE="base" # Use this only for setting up the initial devel image (modify setup_workspace.bash)
+#export DEFAULT_EXECMODE="devel" # This is used while deveoping code and preparing a relase
+#export DEFAULT_EXECMODE="release" # use the release as default
+export DEFAULT_EXECMODE="CD" # use the continuous deployment image as default
```

Comment:
As this repo is the starting point for development, the execmode should be "base" here

Reply:
True. Thanks for spotting this. I did set it to CD for the project specific use case..